src/components/TimeCardPage/TimeTable.tsx

```diff
@@ -26,11 +26,11 @@ const createEmptyRow = (date) => {
 const formatRows = (providedRows, startDate) => {
     const updatedRows = [] 
 
-    var currentDate = moment.unix(startDate); 
+    var currentDate = moment.unix(startDate).tz(TIMEZONE); 
```

Comment:
Oh, this is why you're using `moment`

You cant use ES6 Date objects for this? Dang. 😞

Reply:
We also wanted moment to make duration specific comparisons just alot easier for example comparisons by the day, hour, minute, etc / it has alot of pre-built functionality that allowed us to not have to implement it ourselves such as grabbing the start day of the week, etc.